MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -13,16 +13,24 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func show(message: String)
+	func hide()
+}
+
```

Comment:
Suggestion:
`func show(message: String)` can have optional `message` (`show(message: String?)`). The hiding can be done by calling `show(message: nil)`, because `hide` is a very generic name and can have conflicts with other methods in the future. 
Also it will keep all the state transitions centralized into a single state type (`String?`).

For a single state type It can be better to have a separate `FeedErrorViewModel` that describes the error view state (the same way that `FeedLoadingViewModel` does). It will make introducing new parameters a lot easier. With current approach I will need to modify `FeedErrorView.show` function which will make `FeedPresenter` and `FeedViewController` stop compiling. 

```suggestion
struct FeedErrorViewModel {
	let message: String?
}

protocol FeedErrorView {
	func display(_ viewModel: FeedErrorViewModel)
}
```